addon/components/frost-fixed-table.js

```diff
@@ -335,13 +412,28 @@ export default Component.extend({
 
   // == Lifecycle Hooks =======================================================
 
+  init () {
+    this._super(...arguments)
+    const itemKey = this.get('itemKey')
+    if (itemKey) {
+      this.set('_itemComparator', function (lhs, rhs) {
```

Comment:
This is your default comparison function? Maybe we can isolate this in a util or something? or a function?